code/game/objects/items.dm

```diff
@@ -345,6 +345,29 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 
 			else if(S.can_be_inserted(src))
 				S.handle_item_insertion(src)
+		return 
+	if (istype(user,/mob/living/carbon/human))

```

Comment:
Explain to me how i can make this work without having to make copy pasted code, one for drone's type, one for human's type.

and that alone is untenable, its snowflake and would rightfully be declined by the maintainers.